certified-connectors/KyndrylMainframe/readme.md

```diff
@@ -0,0 +1,31 @@
+# Kyndryl Mainframe Connector
+Unlock the value of your data by using this connector to enable low-code developers to build power apps that will unleash new data insights and capabilities across your organization. Work with the thousands of mainframe developers at Kyndryl to innovate and provide better end user experiences against your mainframe data whilst securing it through Kyndryl Bridge and Microsoft Power Platform. 
+
+## Publisher: Publisher's Name
+Kyndryl
+
+## Prerequisites
+Kyndryl Bridge API Key, Kyndryl Bridge Host URL, Kyndryl Bridge Base URL, IBM mainframe. 
+
+## Supported Operations
+
+### Get Customer
+This action retrieves a specific customers details. 
+
+### Create Customer
+This action creates a new customer profile. 
+
+### Get Policy
+This action retrieves a policy related to a customer. 
+
+### Update Customer
+This action updates a specific customers details. 
+
+## Obtaining Credentials
```

Comment:
@RTkyndryl is it possible to provide some links to the page for obtaining the credentials?